src/js/HtmlUi.js

```diff
@@ -18,7 +20,7 @@ class HtmlUi extends Meister.Ui {
      */
     constructor(config, meister) {
         super(config, meister);
-
+        this.hiddenClassName = this.config.hiddenClassName || 'mstr-hide-controls';
```

Comment:
Not the biggest fan of using class strings without reference, having a source from where the `mstr-hide-controls` came from would be great for developing. Though could be ignored for now and refactored later if we got all styling ready.